### DISCUSSION TITLE
Fonts in Docker image

### DISCUSSION THREAD

**Discussion by trombonehero** [typst/typst#3193] [Ideas]
The official Docker image is very helpful for CI actions, but it only comes with four fonts. Could this image be built with more freely-available fonts (e.g., those shipped with common Linux distros, those from Google Web Fonts) to better show off this typesetting system?

**Comment by alerque** [comment]
This sounds like a bad idea. This project could chase its tail for years trying to add that one more font people wanted and thought should be "common".

Instead I suggest either an easy way to document how to build a custom Docker container that includes other dependencies (fonts or otherwise) or just tips on how to run a Docker container while injecting the system fonts (for example this is [how SILE's Docker container works](https://github.com/sile-typesetter/sile/#docker)). I haven't checked it but if the Docker image you are using is using fontconfig properly you can probably use an adapted version of the command suggested by SILE for mounting your system font path. It may need to be adjusted for where the fonts are expected, something like `docker run -it --volume "/usr/share/fonts:/usr/share/fonts"`.
